packages/core/src/database/find-many.ts

```diff
@@ -21,14 +21,27 @@ export const buildFindMany = <Schema extends SchemaLike, ReturnType extends Sche
       )
       .filter((value): value is Truthy<typeof value> => notFalsy(value));
 
-  return async ({ where, limit, offset }: FindManyData<Schema> = {}) => {
+  const buildOrderBy = (data: OrderBy<Schema>) =>
```

Comment:
This logic seems the same as `connectKeyValueWithEqualSign`.
Add an issue to refactor this?

Reply:
The data type is not the same.
(`Partial<Schema>` and `OrderBy<Schema>`)
If extracted, the implementation is a little complex.

Reply:
so leave an issue for this?

Reply:
It's not a blocker.

Reply:
ok